app/src/main/java/openfoodfacts/github/scrachx/openfood/features/product/view/nutrition/NutritionProductFragment.kt

```diff
@@ -220,6 +220,8 @@ class NutritionProductFragment : BaseFragment(), CustomTabActivityHelper.Connect
             }
         }
 
+        binding.nutriscoreLink.visibility = if (product.nutritionGradeFr != null) VISIBLE else GONE
```

Comment:
```suggestion
        binding.nutriscoreLink.isVisible = product.nutritionGradeFr != null
```
From [core-ktx](https://android.github.io/android-ktx/core-ktx/androidx.view/android.view.-view/is-visible.html)